MeesGame/Gameplay/Content/Content.mgcb

```diff
@@ -223,7 +223,6 @@
 /processorParam:TextureFormat=Color
 /build:fog@16.png
 
-
```

Comment:
Content automatically deletes these whitelines so I think its better if they are gone, they shouldn't even have been there in the first place.